database/insertData/03_template_D_basicReviewTesting.js

```diff
@@ -369,6 +369,38 @@ exports.queries = [
                   }
                 }
               }
+              {
+                actionCode: "updateReviewVisibility"
+                sequence: 2
+                trigger: ON_REVIEW_SUBMIT
+                condition: {
+                  operator: "AND"
+                  children: [
+                    {
+                      operator: "="
+                      children: [
+                        {
+                          operator: "objectProperties"
+                          children: [
+                            "applicationData.reviewData.latestDecision.decision"
```

Comment:
This throws an error ("Can't resolve object") when latestDecision is NULL (as it's trying to find a property that doesn't exist), which it will be if review is Level 1. I've made a new issue (#240) to make the evaluator return Null in cases like this, which will probably help. But is there anything you can do differently in the meantime to avoid this error?

Reply:
I'm about to make this small fix on #240. Do you think handling unresolved objects should be an optional parameter, or should it always return `null`?